package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "paybutton",
+  "version": "2.1.2",
+  "description": "The easiest way to accept eCash online",
+  "author": "poldridge",
+  "main": "index.js",
+  "repository": "https://github.com/PayButton/paybutton.git",
+  "license": "MIT",
+  "scripts": {
+    "build:react":"cd react && yarn && yarn build",
+    "build:paybutton": "yarn clean:paybutton && cd paybutton && yarn && yarn build",
```

Comment:
Why `clean:paybutton` but not `clean:react`? I don't think any should be here, is not always we will want to recompile all packages.

Reply:
because paybutton takes paybutton/react package as a dependecy so It is reasonable to clean the decency after re-installing it

Reply:
Hmm, not sure if I understand but `paybutton/react`, being a file dependency, will go into `node_modules/` as a symbolic link, meaning it will always be a pointer to the up-to-date `../react/` directory

Reply:
Indeed, you're right.

I just found out that `yarn` and `npm` treat differently `file:` dependency resolutions. `npm` would do as I said and create a symbolic link for it; meanwhile yarn actually copies it into `node_modules/`, so changes are not reflected immediately.

That would explain also another issue (I was about to write about) where in my notebook `yarn` takes way  too long to fetch the last package, probably because it's copying it and therefore copying `react/node_modules` itself into `paybutton/node_modules/@paybutton/react/`; a lot of very small files.

Searching about that I found out that this is simply solved by using `link:` instead of `file:` in the `paybutton/package.json`, so unless you see any problem with that I think is the way to go.